### ISSUE TITLE
playwright.cmd does not exist

### ISSUE THREAD

**Issue by AngangGuo** [playwright-community/playwright-go#263]
After I updated to v0.2000.0, I got this error when I run the example code:
```
// go get -u github.com/playwright-community/playwright-go 

pw, err := playwright.Run()
if err != nil {
    log.Fatalf("could not launch playwright: %v", err)
}
```
```
2022/03/15 13:34:29 could not launch playwright: could not start driver: exec: "C:\\Users\\caguoa00\\AppData\\Local\\ms-playwright-go\\1.20.0-beta-1647057403000\\playwright.cmd": file does not exist
```
Please help. Thanks

**Comment by mxschmitt** [comment]
I updated the release notes, there was unfortunately a breaking change included which I did not mention!

See here: https://github.com/playwright-community/playwright-go/releases/tag/v0.2000.0

TL;DR: do playwright.Install() manually and then it should work.

**Comment by AngangGuo** [comment]
Sorry, I didn't go to the release page.
It work great after run this command:
```
go run github.com/playwright-community/playwright-go/cmd/playwright install
```
Great thanks.
